src/transforms/groupby.js

```diff
@@ -35,14 +35,24 @@ exports.attributes = {
         ].join(' ')
     },
     style: {
```

Comment:
That was my template, but I had a hard time tracking down exactly what it *does*. See: [registry.js#L100-L105](https://github.com/plotly/plotly.js/blob/master/src/registry.js#L100-L105). It seems like since `groupby` has no `layoutAttributes`, this key will have no effect.